Accordion.js

```diff
@@ -0,0 +1,496 @@
+/** @module deliteful/Accordion */
+define(["dcl/dcl",
+	"decor/sniff",
+	"requirejs-dplugins/Promise!",
+	"delite/register",
+	"delite/KeyNav",
+	"requirejs-dplugins/jquery!attributes/classes",
+	"delite/DisplayContainer",
+	"./ToggleButton",
+	"delite/theme!./Accordion/themes/{{theme}}/Accordion.css"
+], function (dcl, has, Promise, register, KeyNav, $, DisplayContainer, ToggleButton) {
+
+	function setVisibility(node, val) {
+		node.style.display = val ? "" : "none";
+	}
+
+	var animationEndEvent = (function () {
```

Comment:
Maybe we should add a feature test for animation in order to be able to do `has("animation")` or something like that ?
Some widgets like Viewstack and SidePane only test if the browser is not IE9 to validate if it's possible to perform transitions.
What do you think @wkeese ?